modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -0,0 +1,157 @@
+ * by the title of the object returned from node_load().
+ *
+ * @param $nid
+ *   String of numbers representing the node id passed from hook_menu.
+ * @return
+ *   String of characters to be used as the title in a menu local task for
+ *   policy nodes.
+ */
+function inope_policy_review_tab_title_callback($node) {
+  $node_title = $node->title;
+  $title = t('Review of @node_title', array('@node_title' => $node_title));
+  return $title;
+}
+
+
+/**
+ * Implements hook_permission().
+ */
+function inope_policy_review_permission() {
+  return array(
+    'view inope policy review' => array(
+      'title' => t('View iNope Policy Review'),
+      'description' => t('View the review tab on policy node pages'),
+    ),
+  );
+}
+
+/**
+ * The iNope Policy Review Form.
+ */
+function inope_policy_review_form($form, &$form_state) {
+  global $user;
+
+  $status = array(
+    0 => t('I have reviewed this policy and approved it for use'),
+    1 => t('I have reviewed this policy and would like to see changes'),
+  );
+
+  $form['inope_policy_review_status_radio'] = array(
+    '#type' => 'radios',
+    '#options' => $status,
+  );
+
+  $form['inope_policy_review_changes_requested_textarea'] = array(
+    '#type' => 'textarea',
+    '#title' => t('Changes Requested:'),
+    '#default_value' => '',
+    '#states' => array(
+      'invisible' => array(
+        ':input[name="inope_policy_review_status_radio"]' => array('value' => 0),
+      ),
+      'visible' => array(
+        ':input[name="inope_policy_review_status_radio"]' => array('value' => 1),
+      ),
+      'required' => array(
+        ':input[name="inope_policy_review_status_radio"]' => array('value' => 1),
+      ),
+    ),
+  );
+
+  $form['inope_policy_review_submit'] = array(
+    '#type' => 'submit',
+    '#value' => t('Submit'),
+  );
+  
+  $form['inope_policy_review_hidden_nid'] = array(
+    '#type' => 'hidden',
+    '#value' => $form_state['build_info']['args'][0]->nid,
+  );
+  
+  $form['inope_policy_review_hidden_vid'] = array(
+    '#type' => 'hidden',
+    '#value' => $form_state['build_info']['args'][0]->vid,
+  );
+
+  $form['inope_policy_review_hidden_uid'] = array(
+    '#type' => 'hidden',
+    '#value' => $user->uid,
+  );
+
+  return $form;
+}
+
+/**
+ * The iNope Policy Review Form validate handler.
+ */
+function inope_policy_review_form_validate($form, &$form_state) {
+  if ($form_state['values']['inope_policy_review_status_radio'] != 0 &&
+      $form_state['values']['inope_policy_review_status_radio'] != 1) {
+    form_set_error('inope_policy_review_status_radio', 'Please try again, status radio');
+  }
+  if ($form_state['values']['inope_policy_review_changes_requested_textarea'] !=
+      check_plain($form_state['values']['inope_policy_review_changes_requested_textarea'])) {
+    form_set_error('inope_policy_review_changes_requested_textarea', 'Please try again, only plain text in textarea');
```

Comment:
Again, -- what is the actual issue here -- what is it that isn't allowed -- what would cause the error to fire, give the user something that they can do something - with.  A user might come back with -- I copy and pasted from .... I don't understand what plain text is... etc.